src/test/java/com/baidu/hugegraph/test/functional/AuthRestoreTest.java

```diff
@@ -0,0 +1,332 @@
+                checkGroupAndTarget = true;
+                break;
+            }
+        }
+        Assert.assertTrue(checkGroupAndTarget);
+    }
+
+    @Test
+    public void testAuthRestoreByUser() {
+        this.loadData("./auth-backup/" + HugeType.USER.string(),
+                      "/auth/auth_users.txt");
+
+        String[] args = new String[]{
+                "--test-mode", "true",
+                "--user", USER_NAME,
+                "--password", USER_PASSWORD,
+                "auth-restore",
+                "--types", "user",
+                "--directory", DEFAULT_URL,
+                "--init-password", "123456"
+        };
+
+        HugeGraphCommand.main(args);
+
+        List<User> userList = this.client.auth().listUsers();
+        Map<String, User> userMap = Maps.newHashMap();
+        for (User user1 : userList) {
+             userMap.put(user1.name(), user1);
+        }
+
+        Assert.assertTrue(userMap.containsKey("test_user1"));
+    }
+
+    @Test
+    public void testAuthRestoreWithException() {
+        String[] args = new String[]{
+                "--test-mode", "true",
+                "--user", USER_NAME,
+                "--password", USER_PASSWORD,
+                "auth-restore",
+                "--types", "user",
+                "--directory", DEFAULT_URL
+        };
+
+        Assert.assertThrows(RuntimeException.class, () -> {
+            HugeGraphCommand.main(args);
+        }, (e) -> {
+            String msg = e.getMessage();
+            Assert.assertTrue(msg.endsWith("The following option is required: [--init-password]"));
+            Assert.assertContains("com.beust.jcommander.ParameterException: The " +
+                                  "following option is required: [--init-password]",
+                                  msg);
+        });
+    }
+
+    @Test
+    public void testAuthRestoreByStrategyConflict() {
+        this.loadData("./auth-backup/" + HugeType.USER.string(),
+                      "/auth/auth_users_conflict.txt");
+
+        String[] args = new String[]{
+                "--test-mode", "true",
+                "--user", USER_NAME,
+                "--password", USER_PASSWORD,
+                "auth-restore",
+                "--types", "user",
+                "--strategy", "stop",
+                "--init-password", "123456"
+        };
+
+        Assert.assertThrows(RuntimeException.class, () -> {
+            HugeGraphCommand.main(args);
+        }, (e) -> {
+            String msg = e.getMessage();
+            Assert.assertTrue(msg.startsWith("java.lang.IllegalArgumentException"));
+            Assert.assertContains("java.lang.IllegalArgumentException: " +
+                                  "Restore users conflict with stop strategy",
+                                  msg);
+        });
+    }
+
+    @Test
+    public void testAuthRestoreByStrategyIgnore() {
+        this.loadData("./auth-backup/" + HugeType.USER.string(),
```

Comment:
refactor loadData() to just pass params HugeType.USER and "auth_users_conflict.txt"